counterpartylib/lib/messages/issuance.py

```diff
@@ -164,7 +210,8 @@ def validate (db, source, destination, asset, quantity, divisible, callable_, ca
     if util.enabled('integer_overflow_fix', block_index=block_index) and (fee > config.MAX_INT or quantity > config.MAX_INT):
         problems.append('integer overflow')
 
-    return call_date, call_price, problems, fee, description, divisible, reissuance
+    return call_date, call_price, problems, fee, description, divisible, reissuance, reissued_asset_longname
```

Comment:
This is still necessary.  When a reissuance transaction is parsed, the subasset_longname is **not** passed to validate from `parse`.  Validate looks up the asset_longname and returns it.

It does mix concerns a little bit, but since validate is already looking up the asset, I returned it to avoid duplicate SQL queries.

It could be refactored to be cleaner, but I don't think it is worth it at this time.